tests/Config/Processor/BuilderProcessorTest.php

```diff
@@ -34,28 +36,23 @@ public function testApiAbuse($name, $type, $builderClass, $exceptionClass, $exce
         BuilderProcessor::process($config);
     }
 
-    public function apiAbuseProvider()
+    public function apiAbuseProvider(): array
     {
         return [
-            ['foo', BuilderProcessor::BUILDER_FIELD_TYPE, [], \InvalidArgumentException::class, 'Field builder class should be string, but array given.'],
-            ['foo', BuilderProcessor::BUILDER_FIELDS_TYPE, [], \InvalidArgumentException::class, 'Fields builder class should be string, but array given.'],
-            ['foo', BuilderProcessor::BUILDER_ARGS_TYPE, [], \InvalidArgumentException::class, 'Args builder class should be string, but array given.'],
-            ['bar', BuilderProcessor::BUILDER_FIELD_TYPE, null, \InvalidArgumentException::class, 'Field builder class should be string, but NULL given.'],
-            ['bar', BuilderProcessor::BUILDER_FIELDS_TYPE, null, \InvalidArgumentException::class, 'Fields builder class should be string, but NULL given.'],
-            ['foo', BuilderProcessor::BUILDER_FIELD_TYPE, 'Fake\Foo', \InvalidArgumentException::class, 'Field builder class "Fake\Foo" not found.'],
-            ['foo', BuilderProcessor::BUILDER_FIELDS_TYPE, 'Fake\Foo', \InvalidArgumentException::class, 'Fields builder class "Fake\Foo" not found.'],
-            ['foo', BuilderProcessor::BUILDER_FIELD_TYPE, \stdClass::class, \InvalidArgumentException::class, 'Field builder class should implement "Overblog\GraphQLBundle\Definition\Builder\MappingInterface", but "stdClass" given.'],
-            ['foo', BuilderProcessor::BUILDER_FIELDS_TYPE, \stdClass::class, \InvalidArgumentException::class, 'Fields builder class should implement "Overblog\GraphQLBundle\Definition\Builder\MappingInterface", but "stdClass" given.'],
-            ['foo', BuilderProcessor::BUILDER_ARGS_TYPE, \stdClass::class, \InvalidArgumentException::class, 'Args builder class should implement "Overblog\GraphQLBundle\Definition\Builder\MappingInterface", but "stdClass" given.'],
+            ['foo', BuilderProcessor::BUILDER_FIELD_TYPE, 'Fake\Foo', InvalidArgumentException::class, 'Field builder class "Fake\Foo" not found.'],
+            ['foo', BuilderProcessor::BUILDER_FIELDS_TYPE, 'Fake\Foo', InvalidArgumentException::class, 'Fields builder class "Fake\Foo" not found.'],
+            ['foo', BuilderProcessor::BUILDER_FIELD_TYPE, stdClass::class, InvalidArgumentException::class, 'Field builder class should implement "Overblog\GraphQLBundle\Definition\Builder\MappingInterface", but "stdClass" given.'],
+            ['foo', BuilderProcessor::BUILDER_FIELDS_TYPE, stdClass::class, InvalidArgumentException::class, 'Fields builder class should implement "Overblog\GraphQLBundle\Definition\Builder\MappingInterface", but "stdClass" given.'],
+            ['foo', BuilderProcessor::BUILDER_ARGS_TYPE, stdClass::class, InvalidArgumentException::class, 'Args builder class should implement "Overblog\GraphQLBundle\Definition\Builder\MappingInterface", but "stdClass" given.'],
```

Comment:
Removed entries with `[]` and `null` as third item, because the target method allows only strings anyway (because of type-hint)